Gems/OpenXRVk/Code/Source/InputDeviceXRController.cpp

```diff
@@ -160,11 +180,21 @@ namespace AzFramework
 
         // Connect to haptic feedback request bus
         InputHapticFeedbackRequestBus::Handler::BusConnect(GetInputDeviceId());
+
+#if !defined(_RELEASE)
```

Comment:
Is this the normal pattern for using DebugDisplayEventBus where we need to protect it behind RELEASE macro_? I think it would be cleaner to move the macros within the Bus itself instead of making the calling code messy.

Reply:
I was just wrapping all the "debug" functionality with !_RELEASE because that makes sense to strip it out of release builds.  I can poke a little further and see how DebugDisplayEventBus handles RELEASE (if at all).  I'd prefer to keep this, but we can potentially hide it a bit better.

Reply:
Updated this, let me know what you think.